src/UpdatedPackagesCollector.js

```diff
@@ -52,52 +52,45 @@ export default class UpdatedPackagesCollector {
   collectUpdatedPackages() {
     this.logger.info("", "Checking for updated packages...");
 
-    const hasTags = GitUtilities.hasTags(this.execOpts);
+    const { execOpts, options } = this;
```

Comment:
Boyscout: Instead of accessing on `this` or `options` multiple times, I destructured here.